ui/components/app/transaction-list-item-details/transaction-list-item-details.container.js

```diff
@@ -39,10 +40,12 @@ const mapStateToProps = (state, ownProps) => {
   };
   const rpcPrefs = getRpcPrefsForCurrentProvider(state);
 
+  const networkConfiguration = getNetworkConfigurationsByChainId(state);
   const isCustomNetwork = getIsCustomNetwork(state);
 
   return {
     rpcPrefs,
```

Comment:
Since you removed the this props in the component, we can also remove it from the container.

Reply:
Hmm I also don't see us injecting the `chainId` here. But guess this is a prop that a developer needs to provide then?

Reply:
yes because it's provided by dev